include/fastdds/dds/domain/DomainParticipant.hpp

```diff
@@ -943,6 +946,46 @@ class DomainParticipant : public Entity
      */
     bool has_active_entities();
 
+    /**
+     * Enables the monitor service in this DomainParticipant.
+     *
+     * @return RETCODE_OK if the monitor service could be correctly enabled.
+     * @return RETCODE_ERROR if the monitor service could not be enabled properly.
+     *
+     * @note Not supported yet. Currently returns RETCODE_UNSUPPORTED
+     */
+    RTPS_DllAPI ReturnCode_t enable_monitor_service() const;
+
+    /**
+     * Disable the monitor service in this DomainParticipant. Does nothing if the service was not enabled before.
+     *
+     * @return RETCODE_OK if the monitor service could be correctly disabled.
+     * @return RETCODE_NOT_ENABLED if the monitor service was not previously enabled.
+     * @return RETCODE_ERROR if the service could not be properly disabled.
+     *
+     * @note Not supported yet. Currently returns RETCODE_UNSUPPORTED
+     */
+    RTPS_DllAPI ReturnCode_t disable_monitor_service() const;
+
+    /**
+     * Enables the internal monitor service in this DomainParticipant.
```

Comment:
I think this description can be improved. It is not really describing the method goal which is checking if two provided Proxies are QoS compatible.

Reply:
Totally forgot